cookbooks/bcpc-hadoop/attributes/hbase.rb

```diff
@@ -51,8 +51,8 @@
 default["bcpc"]["hadoop"]["hbase_rs"]["xmn"]["size"] = 256
 default["bcpc"]["hadoop"]["hbase_rs"]["xms"]["size"] = 1024
 default["bcpc"]["hadoop"]["hbase_rs"]["xmx"]["size"] = 1024
-default["bcpc"]["hadoop"]["hbase_rs"]["mx_dir_mem"]["size"] = 256
-default["bcpc"]["hadoop"]["hbase_rs"]["hdfs_dir_mem"]["size"] = 128
+default["bcpc"]["hadoop"]["hbase_rs"]["mx_dir_mem"]["size_by_ratio"] = 0.2 # of total memory
```

Comment:
How is this different from ``default["bcpc"]["hadoop"]["hbase"]["bucketcache"]["size_by_ratio"]``?